index.js

```diff
@@ -165,21 +165,21 @@ async function add({ message, type, branch, silent }) {
             type = answers.type;
         }
 
-        if (!branch) {
-            if (silent) branch = branchNumber;
+        if (!issue) {
+            if (silent) issue = branchNumber;
             else {
-                const answers = await inquirer.prompt(questions.branch);
-            branch = answers.branch;
+                const answers = await inquirer.prompt(questions.issue);
+            issue = answers.issue;
```

Comment:
Missing indentation?

```suggestion
                issue = answers.issue;
```

Reply:
probably, but i will keep it. I'm working on cleaning the code base (eslint + dividing in sub files) and the new PR will also be part of the new incoming version :) It will fixed anyway